nengo_viz/static/viz_slider.js

```diff
@@ -151,6 +127,34 @@ VIZ.Slider = function(args) {
 VIZ.Slider.prototype = Object.create(VIZ.Component.prototype);
 VIZ.Slider.prototype.constructor = VIZ.Slider;
 
+VIZ.Slider.prototype.set_value = function(graph, slider, value) {
```

Comment:
Why is ```graph``` being passed in?  That's just ```this```, after all.  It should be

```
VIZ.Slider.prototype.set_value = function(slider, value) {
```

and called with

```
                   self.set_value(this, 0);
```